### ISSUE TITLE
[android]Migrate Webp to xml assets

### ISSUE THREAD

**Issue by Jean-BaptisteC** [organicmaps/organicmaps#6415]
We have around 1,2 Mo of Webp images on Android, some can be replaced by assets from [Google Fonts](https://fonts.google.com/icons) and others can be migrated to SVG manually.
Some can be reused in different case
- Example: Cancel icon can be removed and use close icon

Using Google Icons is more adapted if in the future we migrate to Material You
I have seen we have a lot of icons for location button animation and loading routing calculation, is it possible to simplify this?

- On Google Fonts, do you want to use Material Icons or Material Symbols? (Material Symbols is more modern and offers more settings before export icons)

@biodranik @rtsisyk @dvdmrtnz @RedAuburn

**Comment by biodranik** [comment]
Reducing the apk size is NOT the main priority now compared to many other issues. Nobody complained so far about some kilobytes of images.

**Comment by Crchbhvtxtv** [comment]
> Reducing the apk size is NOT the main priority now compared to many other issues. Nobody complained so far about some kilobytes of images.

Agree. Maps is heavy itself. It is not important to shrink an app size.

**Comment by Jean-BaptisteC** [comment]
Yes you're right it's not a priority, this work can be doing in same times we migrate png (contains all icons POI) to SVG

**Comment by rtsisyk** [comment]
> Using Google Icons is more adapted if in the future we migrate to Material You I have seen we have a lot of icons for location button animation and loading routing calculation, is it possible to simplify this?
> 
>     * On Google Fonts, do you want to use Material Icons or Material Symbols? (Material Symbols is more modern and offers more settings before export icons)

Maybe even both. It's hard to say before seeing the difference for each particular icon.

> Google Fonts offers two collections as [icon fonts](https://fonts.google.com/knowledge/glossary/icon_font): [Material Icons](https://fonts.google.com/icons) and [Material Symbols](https://fonts.google.com/icons). The Material Symbols [font](https://fonts.google.com/knowledge/glossary/font) is the [variable](https://fonts.google.com/knowledge/glossary/variable_fonts) version of the Material Icons font, combining over 2,000 [glyphs](https://fonts.google.com/knowledge/glossary/glyph) into a single font file with a wide range of design variants, adjusted by variable [axes](https://fonts.google.com/knowledge/glossary/axis_in_variable_fonts).

**Comment by Jean-BaptisteC** [comment]
Some examples of difference:
|Material Symbols|Material Icons|
|-|-|
![search_FILL0_wght400_GRAD0_opsz24](https://github.com/organicmaps/organicmaps/assets/87148630/373feebe-40f1-419e-9a95-577f771d0218)|![search_black_24dp](https://github.com/organicmaps/organicmaps/assets/87148630/883fac4d-907a-4b86-9370-93cf7c25fb46)|
|![settings_FILL0_wght400_GRAD0_opsz24](https://github.com/organicmaps/organicmaps/assets/87148630/1c98f9e1-54d7-4eb4-8604-a7709c38676b)|![settings_black_24dp](https://github.com/organicmaps/organicmaps/assets/87148630/1696305f-2c2d-4495-824b-9e8ccc11423d)

Material Symbols offer more variants of icons and some icons are more simplify than Material Icons

**Comment by Jean-BaptisteC** [comment]
A lot of work has been done here #7226 and #7306 but it's stay somes icons not available in Google fonts website and need to be redraw in SVG

**Comment by biodranik** [comment]
@Jean-BaptisteC which ones? Maybe we can extract them from https://developer.apple.com/sf-symbols/ ?

**Comment by Jean-BaptisteC** [comment]
Icons present here https://github.com/organicmaps/organicmaps/tree/master/android/app/src/main/res/drawable-xxxhdpi
I'm not sure it's a good idea to mix Google Icons with Apple Icons
IMO, somes icons like car or bike icons don't to be update to keep visual identity of OM.

**Comment by biodranik** [comment]
If they look good together, then it's ok. Need to check/compare, of course, if it's better or not.
